cartoframes/data/utils.py

```diff
@@ -0,0 +1,150 @@
+import time
+import binascii as ba
+
+from warnings import warn
+from carto.exceptions import CartoException, CartoRateLimitException
+
+from ..columns import Column
+
+try:
+    import geopandas
```

Comment:
Why do we need that? Why is not a new dependency?

Reply:
This is not a dependency. CARTOframes works without geopandas, but this library is required for some situations. This comes from the past versions, but you can create a ticket to discuss this in future versions of CF.

Reply:
geopandas is a heavy dependency, so I like the idea of making it optional since it's only required for a couple of features. This is a common pattern. E.g., matplotlib is not required for pandas but when installed `dataframe.plot()` returns plots.